script.js

```diff
@@ -268,5 +268,10 @@ $(document).ready(function() {
     
       return 0;
     }
+//page animations
+    $(".panel").slideDown(1500);
```

Comment:
These two animation will not happen again if the user navigates back to the landing page via the navbar brand. They only happen on page load.